main.nf

```diff
@@ -30,6 +19,15 @@ include { METHYLSEQ               } from './workflows/methylseq/'
 include { PREPARE_GENOME          } from './subworkflows/local/prepare_genome/'
 include { PIPELINE_INITIALISATION } from './subworkflows/local/utils_nfcore_methylseq_pipeline'
 include { PIPELINE_COMPLETION     } from './subworkflows/local/utils_nfcore_methylseq_pipeline'
+include { getGenomeAttribute      } from './subworkflows/local/utils_nfcore_methylseq_pipeline'
+
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    GENOME PARAMETER VALUES
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+params.fasta         = getGenomeAttribute('fasta')
+params.bismark_index = getGenomeAttribute('bismark')
```

Comment:
I think this needs to be before any calls to PREPARE_GENOME if I remember well

Reply:
it is before any calls to PREPARE_GENOME